src/main/java/in/nimbo/exception/ElasticException.java

```diff
@@ -0,0 +1,24 @@
+package in.nimbo.exception;
+
+public class ElasticException extends Exception {
```

Comment:
extend RuntimeException instead of exception for being unchecked

Reply:
Fixed